src/parser.ts

```diff
@@ -48,6 +48,19 @@ function handleAttrs(token: Token, type: string) {
         ? { content: token.content }
         : { content: token.content, language };
     }
+    case 'td':
+    case 'th': {
+      const attrs = Object.fromEntries(token.attrs);
+      let align = 'left';
```

Comment:
@rpaul-stripe we probably want the make this `undefined` and then set it to `left` with `:--` right? Just so `left` doesn't always show up in the tree. Or do you prefer it this way?